src/includes/configuration/auto-session-tracking/flutter.mdx

```diff
@@ -1,3 +1,19 @@
+By default, the session is terminated once the application is in the background for more than 30 seconds. You can change the time out with the option named `sessionTrackingIntervalMillis`. It takes the amount in milliseconds. For example, to configure it to be 60 seconds:
+
+```dart
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+Future<void> main() async {
+  await SentryFlutter.init(
+    (options) => options.autoSessionTrackingIntervalMillis = 10000, // it's 30000 milliseconds (30s) by default
+    appRunner: () => runApp(MyApp()),
+  );
+}
```

Comment:
same as https://github.com/getsentry/sentry-docs/pull/2743/files#r536377858

I'd also remove the comment `// it's 30000 milliseconds (30s) by default`
as the text already mentions that

Reply:
Fixed

Reply:
guess you forgot to commit this one